<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>event_inter</class>
 <widget class="QMainWindow" name="event_inter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1111</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/backround/49288c02e77c69aba39e0de2615dd6e4.jpg);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter Evenement</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>50</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 204, 142);
border-radius:15px;

</string>
      </property>
      <property name="text">
       <string>ID_Evenement</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>DATE_E</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Depense</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TYPE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>THEME</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditID_Evenement">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(224, 255, 172);
border-color: rgb(227, 192, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Depense">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_THME">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>80</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="Date_E_Edit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="lineEdit_TYPE">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>201</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Birthday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mariage</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Engament</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Babyshower</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Brideparty</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Autre</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher Evenement</string>
     </attribute>
     <widget class="QTableView" name="afficher">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>721</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher_button">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>rechercher</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>280</y>
        <width>191</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Trier par Type</string>
      </property>
      <widget class="QRadioButton" name="radioButton_croissant">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri Croissant</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_dcroissant">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri decroissant</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="id_reparation">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>90</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Id_Evenement</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>380</y>
        <width>256</width>
        <height>190</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>460</y>
        <width>321</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>476</y>
        <width>47</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Total :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_total">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>476</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_calculer">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>480</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Supprimer Evenement</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Evenement_ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditID_SUPP">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>250</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Modifier Evenement</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID_EVENEMENT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>DATE_E</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ID_EVENEMENTL_modif">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>110</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>80</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Statistique</string>
     </attribute>
     <widget class="QPushButton" name="statsmed">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>-50</y>
        <width>301</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <underline>true</underline>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Appliquer Statistique</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>imprimer</string>
     </attribute>
     <widget class="QPushButton" name="btn_imp">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>0</y>
        <width>261</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string>imprimer</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1123</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
